docs/index.rst

```diff
@@ -24,6 +24,7 @@ and development.
 
    baseplate-healthcheck: Is your service alive? <cli/healthcheck>
    baseplate-serve: The application server <cli/serve>
+   baseplate-script: Run offline jobs <cli/script>
```

Comment:
Yeah, it could. I was trying to explain that it's for non-request workflows. How about "backend"? "unservery"? (unsavory? :P) "Run scripts"?